public/js/app/controllers/TimelineGenericController.js

```diff
@@ -177,6 +177,10 @@ define(["config",
                 }
               })
             }
+          }).catch(function(error) {
+            if (error.responseJSON.err == 'Maximum post-length is 1500 graphemes') {
+              alert("Posts longer than 1500 symbols are not allowed");
```

Comment:
In the latest revision we have this:
```
this.displayMessage('Posts longer than 1500 symbols are not allowed')
```
where `this` is inherited from ApplicationController.